### PR TITLE
Expose createAnalyticsSDRF and createSummarizedExperiment

### PR DESCRIPTION
**Pull request by pcm32**
This PR exposes two additional methods for external consumption: createAnalyticsSDRF & createSummarizedExperiment.

### CODE REVIEW

File: R/functions.R [hunk 1]
```diff
@@ -188,7 +188,7 @@ summarizeAtlasExperiment <- function( experimentAccession, atlasExperimentDirect
 	atlasExperimentSummary <- lapply( allAnalytics, function( analytics ) {
 		
 		# Get the SDRF rows for this analytics object's assays.
-		analyticsSDRF <- .createAnalyticsSDRF( analytics, atlasSDRF )
```
Comment (suhaibMo) [inline R/functions.R:191]: Ah, your making the hidden `.` prefix function to normal one to export this function

File: .github/workflows/Rbuild.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+    - name: R Build and Checks
+      uses: Swechhya/R-actions@v1.1
+      with:
+        action: 'all'
```
Comment (suhaibMo) [inline .github/workflows/Rbuild.yml:12]: R builds and checks tests are failing in Github. Do you still include this file?

**Comment by suhaibMo** [comment]
Overall looks good to me @pcm32 . Only concerned about the Github checks.

**Comment by pcm32** [comment]
So for the test to pass, the writer of the github actions (external) would need to install bioconductor I think to the environment. On my machine, through devtools, it builds and install fine @suhaibMo.

**Comment by suhaibMo** [comment]
OK I get it. Then we to wait until the writer installs Bioconductor as checks will not pass potentially. Can we remove Github checks as it not serving our tests at this occasion?

**Review by suhaibMo — APPROVED** [review]

**Pull request merged**